composer.json

```diff
@@ -22,8 +22,8 @@
   },
   "require": {
     "php": "^7.1 | ^8.0",
-    "symfony/console": "~2.0 || ^3.0 || ^4.0",
-    "symfony/finder": "~2.0 || ^3.0 || ^4.0"
+    "symfony/console": "~2.0 || ^3.0 || ^4.0 || ^5.0",
+    "symfony/finder": "~2.0 || ^3.0 || ^4.0 || ^5.0"
```

Comment:
I think version 2 can be removed as it reached end of life.

Reply:
Handled in https://github.com/malukenho/docheader/commit/a15c22ba8a44add72a2b2bca5e40af244a53497c 🎉